kura/org.eclipse.kura.rest.identity.provider/OSGI-INF/IdentityRestServiceV1.xml

```diff
@@ -43,18 +45,18 @@
                name="UserAdmin"
                policy="static" />
                
-    <reference interface="org.eclipse.kura.internal.rest.identity.provider.IdentityService" 
-               bind="bindIdentityService" 
+    <reference interface="org.eclipse.kura.internal.rest.identity.provider.LegacyIdentityService" 
+               bind="bindLegacyIdentityService" 
                cardinality="0..1"  
-               name="IdentityService" 
+               name="LegacyIdentityService" 
                policy="static" />
 
     <service>
-        <provide interface="org.eclipse.kura.internal.rest.identity.provider.IdentityRestService"/>
+        <provide interface="org.eclipse.kura.internal.rest.identity.provider.IdentityRestServiceV1" />
```

Comment:
Should we export this as a service in the framework? If not, is the service exported in `IdentityRestService.xml` needed as well? I think we should consider removing `IdentityService.xml` DS specification and export here `org.eclipse.kura.internal.rest.identity.provider.IdentityRestService`, if necessary.

Reply:
it's not an issue since they are not api, similar services are exposed in others rest bundle